ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml

```diff
@@ -57,19 +57,19 @@ operator:
   bundles:
   - dockerfile_path: bundleci.Dockerfile
   substitutions:
-  - pullspec: quay.io/openshift/origin-sriov-dp-admission-controller:4.8
+  - pullspec: quay.io/openshift/origin-sriov-dp-admission-controller:4.9
```

Comment:
Shall we use the `latest` for the master branch?

Reply:
I think not, since the csv of the master branch is the one of release-4.9, which references the 4.9 tag 
https://github.com/openshift/sriov-network-operator/blob/5c2581ddd7843433f4cd3ce80938f766d49326fd/manifests/4.9/sriov-network-operator.clusterserviceversion.yaml#L283

Reply:
I'm not sure, if it helps and I'm not wrong after looking at some other configurations I see that they specify 4.9 and not master/latest as well.